### ISSUE TITLE
Transaction Too Large Exception

### ISSUE THREAD

**Issue by arsalan1212** [massivemadness/Squircle-CE#83]
Hello, I am getting Transaction Too Large Exception on **JSON** file having **500 records**. Here is the logcat Exception  message

```
 **java.lang.RuntimeException: android.os.TransactionTooLargeException: data parcel size 1679224 bytes**
        at android.view.autofill.AutofillManager.startSessionLocked(AutofillManager.java:844)
        at android.view.autofill.AutofillManager.notifyViewEntered(AutofillManager.java:467)
        at android.view.autofill.AutofillManager.notifyViewEntered(AutofillManager.java:446)
        at android.view.View.notifyEnterOrExitForAutoFillIfNeeded(View.java:6980)
        at android.view.View.onWindowFocusChanged(View.java:12071)
        at android.widget.TextView.onWindowFocusChanged(TextView.java:9858)
        at android.widget.AutoCompleteTextView.onWindowFocusChanged(AutoCompleteTextView.java:1099)
        at android.view.View.dispatchWindowFocusChanged(View.java:12041)
        at android.view.ViewGroup.dispatchWindowFocusChanged(ViewGroup.java:1402)
        at android.view.ViewGroup.dispatchWindowFocusChanged(ViewGroup.java:1402)
        at android.view.ViewGroup.dispatchWindowFocusChanged(ViewGroup.java:1402)
        at android.view.ViewGroup.dispatchWindowFocusChanged(ViewGroup.java:1402)
        at android.view.ViewGroup.dispatchWindowFocusChanged(ViewGroup.java:1402)
        at android.view.ViewGroup.dispatchWindowFocusChanged(ViewGroup.java:1402)
        at android.view.ViewGroup.dispatchWindowFocusChanged(ViewGroup.java:1402)
        at android.view.ViewRootImpl$ViewRootHandler.handleMessage(ViewRootImpl.java:4184)
        at android.os.Handler.dispatchMessage(Handler.java:108)
        at android.os.Looper.loop(Looper.java:166)
        at android.app.ActivityThread.main(ActivityThread.java:7529)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:245)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:921)
```

**Comment by massivemadness** [comment]
@arsalan1212 could you send me this file? 🤔 I'll try to reproduce this

**Comment by arsalan1212** [comment]
@massivemadness are you talking about Json File?

**Comment by massivemadness** [comment]
@arsalan1212 yes, I want to check this on my device

**Comment by arsalan1212** [comment]
Github doesn't support json file, so sending json file content in txt file.

[Sample Json File with 500 Records.txt](https://github.com/massivemadness/Squircle-IDE/files/6789649/Sample.Json.File.with.500.Records.txt)

**Comment by arsalan1212** [comment]
@massivemadness This issue is occurring in android lower version like android 8, so kindly test it on API level below 27. I have also tested it on android Q and android R and are working on it.

**Comment by uxen-lb** [comment]
> @massivemadness This issue is occurring in android lower version like android 8, so kindly test it on API level below 27. I have also tested it on android Q and android R and are working on it.

Hello, i am having the same issue with different devices (android version 8.1.0, 7.0.0) can you explain why the error occurs please ? 
Also, how to can i identify the source or process that pushes my application to throw this exception.
Thank you in advance

**Comment by massivemadness** [comment]
Fixed in [v2023.1.0](https://github.com/massivemadness/Squircle-CE/releases/tag/2023.1.0)
